language-extensions/dotnet-core-CSharp/README.md

```diff
@@ -0,0 +1,48 @@
+# .NET Core CSharp Language Extension
+
+## Getting Started
+Language Extensions is a feature of SQL Server used for executing external code. The relational data can be used in the external code using the extensibility framework.
+
+For more information about SQL Server Language Extensions, refer to this [documentation](https://docs.microsoft.com/en-us/sql/language-extensions/language-extensions-overview?view=sql-server-ver15).
+
+The dotnet-core-CSharp-extension version in this repository is compatible with SQL Server 2019 CU3 onwards. It integrates .NET core in SQL Server and works with .NET 5.0 in **Windows only**.
+
+Currently, the extension supports the following data types: SQL_C_SLONG, SQL_C_ULONG, SQL_C_SSHORT, SQL_C_USHORT, SQL_C_SBIGINT, SQL_C_UBIGINT, SQL_C_STINYINT, SQL_C_UTINYINT, SQL_C_BIT, SQL_C_FLOAT, SQL_C_DOUBLE, SQL_C_CHAR. It supports the following SQL data types: int, bigint, smallint, tinyint, real, float, bit, varchar(n).
+
+To use this dotnet-core-CSharp-lang-extension.zip package, follow [this tutorial](./sample/regex/README.md). For any fixes or enhancements, you are welcome to modify, rebuild and use the binaries using the following instructions.
+
+## Building
+
+### Windows
+1. Install latest [Visual Studio](https://visualstudio.microsoft.com/vs/) selecting the following required workloads: ".NET desktop development", "Desktop Development with C++" and ".NET core cross platform".
+
+2. Run [build-dotnet-core-CSharp-extension.cmd](./build/windows/build-dotnet-core-CSharp-extension.cmd) which will generate: \
+        - PATH\TO\ENLISTMENT\build-output\dotnet-core-CSharp-extension\windows\debug\nativecsharpextension.dll \
```

Comment:
We should make sure end-users build this in release mode. So, please update the build configuration to make sure by default, it builds in the release.